.circleci/config.yml

```diff
@@ -24,13 +24,23 @@ orbs:
 jobs:
   test-packages:
     <<: *node_executor
+    resource_class: medium+
```

Comment:
Can we remove this back to default and see if tests still pass?

Reply:
Tried to remove this [here](https://github.com/bigcommerce/checkout-js/pull/2016/commits/9f5504efbfb506d8145343421a6dbee062e44580), the test-packages does not pass, hence reverting it.